error: 

   ┌── tests/move_check/parser/extra_text_after_main.move:5:1 ───
   │
 5 │ foo() {
   │ ^^^ Unexpected text after end of main function
   │

